hw11/states/main.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <locale.h>
+#include <stdlib.h>
+
+#include "test.h"
+#include "states.h"
+#include "readFromFile.h"
+
+int main(void) {
+    setlocale(LC_ALL, "RUS");
+    if (!test()) {
+        printf("\n\nTest failed!\n\n");
+        return -1;
+    }
+    int (*dist)[CITIES] = malloc(CITIES * sizeof * dist);
+    if (dist == NULL) {
+        return -1;
+    }
+
+    int visited[100] = { 0 };
+    int capitals[100];
+
+    int n = 0;
+    int m = 0;
+    int k = 0;
+
+    const char filename[100] = "file.txt";
```

Comment:
```suggestion
    const char filename[] = "file.txt";
```
Вам же не важно, что именно 100